packages/devtools_app/web/index.html

```diff
@@ -72,51 +72,69 @@
     }
 
     if ('serviceWorker' in navigator) {
-      // Service workers are supported. Use them.
-      window.addEventListener('load', function () {
-        // Wait for registration to finish before dropping the <script> tag.
-        // Otherwise, the browser will load the script multiple times,
-        // potentially different versions.
-        var serviceWorkerUrl = 'service_worker.js?v=' + version;
-        navigator.serviceWorker.register(serviceWorkerUrl)
-          .then((reg) => {
-            function waitForActivation(serviceWorker) {
-              serviceWorker.addEventListener('statechange', () => {
-                if (serviceWorker.state == 'activated') {
+      if (window.location.hostname === 'localhost' ||
```

Comment:
checking for localhost doesn't seem safe. Could easily be running devtools from localhost in a release build.

Reply:
SG, modified to just check for `index.ddc.html`